src/arg.rs

```diff
@@ -0,0 +1,64 @@
+use clap::{Arg, Command};
```

Comment:
Question: Is there ea reason you prefer the clap's arg command to the one from rust's library?

Reply:
I'm not very familiar with rust's standard library but clap gives us more flexibility